### DISCUSSION TITLE
Cannot figure out how to resize images.

### DISCUSSION THREAD

**Discussion by AlxLhrNc** [academicpages/academicpages.github.io#2275] [Q&A] [answered]
Hi,

Apologies if it has been addressed, but I cannot manage to resize images correctly. Or at least, the resizing is not displayed properly on the webpage form. 

In the `.md` (https://github.com/AlxLhrNc/AlxLhrNc.github.io/blob/master/_portfolio/2024_EcoTaxa.md), the display is as wished, using the `<img "image.jpg" width=auto height="180">` HTML format recommended on Github.

But in the webpage format (https://alxlhrnc.github.io/portfolio/2024_EcoTaxa/), the images are in there original sizes. 

I tried editing the `_config.yml` with `markdown: kramdown` (line 184), and commenting `width: 100%;` in `_base.scss`.

Anything I missed that would force display of the images in native sizes on webpage format?

Thanks in advance.

**Comment by rjzupkoii** [comment]
The template doesn't do much by way of changing the size of images in your content, but the [usual HTML attributes](https://www.w3schools.com/tags/att_img_width.asp) should work fine for adjusting the size of the images. In my own site I also just have two different image sizes in some cases since image editing software does a better job of resizing compared to most web browsers.

**Reply by AlxLhrNc** [reply]
Thanks for answering.

So your suggestion is more "editing the image size prior to upload" rather than actually using the HTML attributes which, as stated in my case, do not work. 

Am I understanding you correctly?

**Reply by rjzupkoii** [reply, marked as answer]
@AlxLhrNc Yes, although if you are using the `img` HTML tags then you should still be able to adjust the height and width via the tag attributes (or CSS for that matter) since the template doesn't make any changes to HTML tags. However, it's generally good practice to keep images files only as large as the need to be for display on the page, and the allow the user to open the larger version on a new page / download, since this helps the page load as quickly as possible and reduces the amount of bandwidth used.

**Reply by AlxLhrNc** [reply]
Ok thanks. 
It doesn't really fix the `img` tag apparently not working in my case, but I'll consider the subject answered, and look why my fork seems broken for that particular tag. 
Thanks again!
